README.md

```diff
@@ -170,15 +170,7 @@ resources { 'firewallchain':
 Internal chains can not be deleted. In order to avoid all the confusing
 Warning/Notice messages when using `purge => true`, like these ones:
 
```

Comment:
can you add a migration path to the README.md / document the breaking changes?

Reply:
Will do, have a blog post I can adapt so shouldn't take long.

Reply:
Have added